marathon_lb.py

```diff
@@ -682,14 +714,31 @@ def generateHttpVhostAcl(templater, app, backend):
                         backend=backend
                     )
                 else:
-                    https_frontend_acl = templater.\
-                        haproxy_https_frontend_acl(app)
-                    staging_https_frontends += https_frontend_acl.format(
-                        cleanedUpHostname=acl_name,
-                        hostname=vhost_hostname,
-                        appId=app.appId,
-                        backend=backend
-                    )
+                    if haproxy_map:
+                        if 'map_https_frontend_acl' not in duplicate_map:
+                            https_frontend_acl = templater.\
+                                haproxy_map_https_frontend_acl(app)
+                            staging_https_frontends += https_frontend_acl.\
+                                format(
+                                    hostname=vhost_hostname,
+                                    haproxy_dir="/".
```

Comment:
Thats a line break for pep8 , 79 character limit, its actually

staging_https_frontends += https_frontend_acl.\
                                format(
                                    hostname=vhost_hostname,
                                    haproxy_dir="/".
                                    join(config_file.split("/")[:-1])
                                )